app/manager-frontend/package.json

```diff
@@ -36,6 +36,7 @@
     "react-router-dom": "^6.2.1",
     "remark-gfm": "^3.0.1",
     "swr": "^1.2.0",
+    "vite-plugin-svgr": "^2.2.2",
```

Comment:
can this be added as a dev dependency instead?

Reply:
yep, its done now.